bundles/org.openhab.binding.teslascope/src/main/java/org/openhab/binding/teslascope/internal/TeslascopeHandler.java

```diff
@@ -0,0 +1,381 @@
+                            ac(true);
+                        } else {
+                            ac(false);
+                        }
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_CHARGE:
+                    if (command instanceof OnOffType onOffCommand) {
+                        if (onOffCommand == OnOffType.ON) {
+                            charge(true);
+                        } else {
+                            charge(false);
+                        }
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_CHARGE_PORT:
+                    if (command instanceof OnOffType onOffCommand) {
+                        if (onOffCommand == OnOffType.ON) {
+                            chargeDoor(true);
+                        } else {
+                            chargeDoor(false);
+                        }
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_DOOR_LOCK:
+                    if (command instanceof OnOffType onOffCommand) {
+                        if (onOffCommand == OnOffType.ON) {
+                            lock(true);
+                        } else {
+                            lock(false);
+                        }
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_FLASH_LIGHTS:
+                    if (command instanceof OnOffType onOffCommand) {
+                        flashLights();
+                        return;
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_FRONT_TRUNK:
+                    if (command instanceof OnOffType onOffCommand) {
+                        openFrunk();
+                        return;
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_HONK_HORN:
+                    if (command instanceof OnOffType onOffCommand) {
+                        honkHorn();
+                        return;
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_REAR_TRUNK:
+                    if (command instanceof OnOffType onOffCommand) {
+                        openTrunk();
+                        return;
+                    }
+                    break;
+                case TeslascopeBindingConstants.CHANNEL_SENTRY_MODE:
+                    if (command instanceof OnOffType onOffCommand) {
+                        if (onOffCommand == OnOffType.ON) {
+                            sentry(true);
+                        } else {
+                            sentry(false);
+                        }
+                    }
+                    break;
+            }
+            logger.debug("Received command ({}) of wrong type for thing '{}' on channel {}", command,
```

Comment:
this will always be called, i guess you meant to add this to a default block ?

Reply:
Yeah  I guess, my command block was loosely based on https://github.com/psmedley/openhab-addons/blob/main/bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinBaseHandler.java#L136 -  I think I've  fixed now.